### PR TITLE
✨Source Lemlist: Make Connector Compatible with Builder

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
* Make the connector compatible with the builder

## How
* Import schema to manifest
* Update docs, 
* Completely made compatible with the builder

## User Impact
* No impact, chore for maintainability.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚

### CODE REVIEW

**Comment by btkcodedev** [comment]
CI is green, Ready to merge
image

**Comment by btkcodedev** [comment]
The auth scheme could be Basic auth with `username` as empty and `password` as api_key 
OR we can pass api_key with access_token query param.
Builder doesn't support empty usernames, thus choosing the second method,

No breaking changes, all works the same :white_check_mark:

**Comment by btkcodedev** [comment]
Removed $parameter macro :white_check_mark: 
Imported Schema :white_check_mark: 
GitHub CI Green  :white_check_mark:

File: docs/integrations/sources/lemlist.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ The Lemlist connector should not run into Lemlist API limitations under normal u
 
 | Version | Date       | Pull Request                                             | Subject                  |
 | :------ | :--------- | :------------------------------------------------------- | :----------------------- |
+| 0.2.2   | 2024-05-30 | [38119](https://github.com/airbytehq/airbyte/pull/38119) | Add builder compatability  |
```
Comment (girarda) [inline docs/integrations/sources/lemlist.md:40]:
```suggestion
| 0.2.2   | 2024-05-13 | [38119](https://github.com/airbytehq/airbyte/pull/38119) | Add builder compatability  |
```

**Review by girarda — APPROVED** [review]
let's first fix the date in the changelog, then 🚢

**Comment by btkcodedev** [comment]
@girarda Suggested change done

**Pull request merged**